components/brave_shields/browser/brave_shields_web_contents_observer.cc

```diff
@@ -100,6 +100,9 @@ BraveShieldsWebContentsObserver::BraveShieldsWebContentsObserver(
 
 void BraveShieldsWebContentsObserver::RenderFrameCreated(
     RenderFrameHost* rfh) {
+  if (!rfh->IsRenderFrameLive())
```

Comment:
we just went through this in muon - see https://github.com/brave/muon/commit/56d63560939ef407eaddea685c235b5b166b1475
and all those calls to `extensions::ExtensionApiFrameIdMap` should be removed